src/hdf5/cloudsat_netcdf_to_ioda.py

```diff
@@ -0,0 +1,162 @@
+#!/home/imoradi/packages/anaconda3/bin/python3.4
+# 2023, Isaac Moradi
+
+import sys
+import netCDF4 as nc
+import xarray as xr
+import numpy as np
+import datetime
+import time
+import read_cloudsat
+
+# fname_in = ['2009335004815_19116_CS_2B-GEOPROF_GRANULE_P1_R05_E02_F00.hdf']
+fname_in = ['2009212223327_17338_CS_2B-GEOPROF_GRANULE_P1_R05_E02_F00.hdf']
+fname_out = "cpr_cloudsat_obs_2009335.nc4"
+
+cpr_obs = read_cloudsat.read_cloudsat(fname_in)
+cpr_obs["sequenceNumber"] = xr.DataArray(np.arange(cpr_obs.obs_id.size), cpr_obs.obs_id.coords)
```

Comment:
@imoradi would be good to discuss this with you

why isn't this particular specification of `sequenceNumber` in the `read_cloudsat`?   But think i've got a handle on this and would like to pick it up if you're OK with that